sdk/core/core-rest-pipeline/src/pipelineRequest.ts

```diff
@@ -68,6 +68,16 @@ export interface PipelineRequestOptions {
    */
   streamResponseStatusCodes?: Set<number>;
 
+  /**
+   * BROWSER ONLY
+   *
+   * A browser only option to enable browser Streams. If this option is set and a response is a stream
+   * the response will have a property `browserStream` instead of `blobBody` which will be undefined.
```

Comment:
```suggestion
   * A browser only option to enable use of the Streams API. If this option is set and streaming is used
   * (see `streamResponseStatusCodes`), the response will have a property `browserStream` instead of 
   * `blobBody` which will be undefined.
```